locale/templates/LC_MESSAGES/amo.pot

```diff
@@ -514,7 +514,7 @@ msgid "No add-ons found"
 msgstr ""
 
 #: src/amo/components/SearchContextCard/index.js:45
-msgid "%(count)s add-on found\""
+msgid "%(count)s add-on found"
```

Comment:
You shouldn't need to modify/commit this file. We'll do that as part of our locale extract/merge. Just changing the source should be fine.